pattern-1/README.md

```diff
@@ -0,0 +1,94 @@
+# DC/OS Resources for WSO2 API Manager Deployment Pattern 1
```

Comment:
Shall we brake this into two lines after "API Manager" for better viewability ?

Reply:
Thanks for the feedback Dilan! I think the convention of writing titles is to write them in one line and let the Markdown interpreter to format that accordingly.

Reply:
I had a look on the actual page on how it looks. This was a suggestion made based on that.

Reply:
Thanks Dilan! This is now updated!